tests/phpunit/api/v3/ContributionTest.php

```diff
@@ -2485,6 +2485,61 @@ public function testRepeatTransactionUpdatedCampaign() {
     ));
   }
 
+  /**
+   * CRM-20685 Repeattransaction produces incorrect Financial Type ID (in specific circumstance) - if number of lineItems = 1.
+   *
+   * This case happens when the line item & contribution do not have the same type in his initiating transaction.
+   */
+  public function testRepeatTransactionUpdatedFinancialTypeAndNotEquals() {
+    $originalContribution = $this->setUpRecurringContribution(array(), array('financial_type_id' => 2));
+    // This will made the trick to get the not equals behaviour.
+    $EWSUpdateLineItem = civicrm_api3('line_item', 'create', array('id' => 1, 'financial_type_id' => 4));
```

Comment:
minor thing but 'EWS' is probably not going to be meaningful to people in the future

Reply:
I will change it today :)